src/foam/lib/json/StringParser.java

```diff
@@ -9,68 +9,19 @@
 import foam.lib.parse.PStream;
 import foam.lib.parse.Parser;
 import foam.lib.parse.ParserContext;
+import foam.lib.parse.Alt;
 
 public class StringParser
   implements Parser
```

Comment:
Couldn't this extend AltParser and configure in consturctor rather than every time in parse() method.

Reply:
It could! I prefer this method since delegation is a tad more readable and natural than extension here. I will move the initialization of the alt object to the body to address the performance issue.